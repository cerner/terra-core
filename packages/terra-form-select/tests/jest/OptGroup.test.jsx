import React from 'react';
/* eslint-disable import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import OptGroup from '../../src/shared/_OptGroup';

describe('OptGroup', () => {
  it('should render a default Option', () => {
    const wrapper = shallowWithIntl(<OptGroup label="label" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should return true for isOptGroup', () => {
    const option = <OptGroup label="label" />;
    expect(option.type.isOptGroup).toBeTruthy();
  });
});
