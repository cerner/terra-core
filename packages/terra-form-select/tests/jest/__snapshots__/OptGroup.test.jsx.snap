// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptGroup should render a default Option 1`] = `
<li
  className="opt-group"
>
  <div
    className="label"
  >
    label
  </div>
  <ul
    className="options"
  />
</li>
`;
